pep-0570.rst

```diff
@@ -494,7 +541,44 @@ A function definition which makes use of both of these features may look like::
            |                                - Keyword only
             -- Positional only
 
-Consider the following example functions::
+where ``/`` and ``*`` are optional. If used, these symbols indicate the kind of
+parameter by how the arguments may be passed to the function:
+positional-only, positional-or-keyword, and keyword-only. Keyword parameters
+are also referred to as named parameters.
+
+-------------------------------
+Positional-or-Keyword Arguments
+-------------------------------
+
+If ``/`` and ``*`` are not present in the function definition, arguments may
+be passed to a function by position or by keyword.
+
+--------------------------
+Positional-only Parameters
+--------------------------
+
+Looking at this in a bit more detail, it is possible to mark certain parameters
+as *positional-only*. If *positional-only*, the parameters' order matters, and
+the parameters cannot be passed by keyword. By placing a ``/`` (forward-slash) in
+the parameters list after the last positional-only parameter, a developer is
+logically separating positional-only parameters from the rest of the parameters.
```

Comment:
IMO the key point here is that positional-only parameters come *before* the `/` (and if there's no `/`, there are no pos-only params). IOW I'd emphasize that the pos-only params come *before* the `/`, rather than stating that the `/` comes after the pos-only params.